lib/pure/os.nim

```diff
@@ -3300,3 +3300,89 @@ func isValidFilename*(filename: string, maxLen = 259.Positive): bool {.since: (1
     find(f.name, invalidFilenameChars) != -1): return false
   for invalid in invalidFilenames:
     if cmpIgnoreCase(f.name, invalid) == 0: return false
+
+
+since (1, 5):
+  let osTrash* {.noWeirdTarget.}: string =
```

Comment:
This is a top level let with side effects, in a very common stdlib module. Shouldn't this be a proc?